core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -218,6 +221,128 @@
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_SERVER_METHOD));
 
+  // Interval Stats
+  static final Duration MINUTE = Duration.create(60, 0);
+  static final Duration HOUR = Duration.create(60 * 60, 0);
+
+  /**
+   * Census defined rpc client {@link IntervalViewDescriptor}s.
+   */
+  public static final IntervalViewDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client roundtrip latency",
+          "Latency in msecs",
```

Comment:
These definitions are consistent with what's defined internally. This will need to be resolved as part of the resolution of 88.